Production/govuk_ios/Extensions/UIKit/URLOpening.swift

```diff
@@ -0,0 +1,11 @@
+import UIKit
+
+protocol URLOpening {
+    func canOpenURL(_ url: URL) -> Bool
+    func open(_ url: URL,
+              options: [UIApplication.OpenExternalURLOptionsKey: Any],
+              completionHandler completion: ((Bool) -> Void)?
+    )
```

Comment:
Can you knock this up to line 7, please? I know this seems a little off from other areas, I can talk you though the madness though

Reply:
Done